lib/features/manage_account/presentation/mailbox_visibility/mailbox_visibility_view.dart

```diff
@@ -1,10 +1,4 @@
 import 'package:core/core.dart';
-import 'package:core/presentation/extensions/color_extension.dart';
```

Comment:
why remove it?

Reply:
I didn't look closely, I should have removed `import 'package:core/core.dart';` instead of deleting the details